shared/components/StudyInfo/Details.tsx

```diff
@@ -330,66 +330,114 @@ const TodentamisenProsessiCollapsed = ({
   todentamisenProsessi?: TodentamisenProsessi
 }) => (
   <>
-    <TodentamisenProsessiSuoraan
-      todentamisenProsessiKoodi={todentamisenProsessiKoodi}
-    />
+    {todentamisenProsessiKoodi === TodentamisenProsessiKoodi.SUORAAN && (
```

Comment:
`=> todentamisenProsessiKoodi === TodentamisenProsessiKoodi.SUORAAN && (`

Reply:
I think there needs to be react.fragment as there is also todentamisenProsessiKoodi ===  TodentamisenProsessiKoodi.ARVIOIJIEN_KAUTTA  ?